### PR TITLE
Build IGM systems in IGMSightline

### PR DESCRIPTION
**Pull request by profxj**
As titled

### CODE REVIEW

**Review by jnburchett — APPROVED** [review]
Looks great. Merging

File: pyigm/igm/igmsightline.py [hunk 1]
```diff
@@ -10,6 +10,7 @@
 
 from linetools.isgm.abssightline import AbsSightline
```
Comment (jnburchett) [inline pyigm/igm/igmsightline.py:11]: Tests are failing here?

**Review by jnburchett — APPROVED** [review]
Oops, just noticed that a couple of the tests are failing on line 11.  No idea why, though, because this import is totally valid and works on my end.  I'm fine with merging if we are okay with the test failing.

File: pyigm/igm/igmsightline.py (hunk 1 above)
Comment (profxj) [reply pyigm/igm/igmsightline.py:11]: This isn't showing on Travis.  I suspect you need to update linetools

**Comment by profxj** [comment]
Merging

**Pull request merged**